components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -55,6 +55,8 @@ spec:
             value: "128"
          {{ if has "hybrid" .Values.pump.backend }}
           # Configure Hybrid Pump for Tyk Pump
+          # If you want to use hybrid pump with MDCB please also set
```

Comment:
It will be useful to move this comment to values file

Reply:
Fixed